### ISSUE TITLE
SKS coverage

### ISSUE THREAD

**Issue by cfcs** [robur-coop/ocaml-openpgp#19]
It would be cool to get an overview of "real world" coverage, for example using the dumps from https://pgp.key-server.io/sks-dump/

I expect a good portion of these to be broken, and there are also some previously valid keys that are intentionally not supported in this library (El-Gamal signing keys, <1024 bit keys, ...), but nevertheless some kind of overview (categorization, perhaps a pie chart) would be nice.

**Comment by lambdafu** [comment]
Yes. There are some older results here: "Investigating the OpenPGP Web of Trust" (Ulrich, Holz, Hauck, Carle), in particular section 6. I also wrote a script to parse gpg --list-packet, but have stalled to write my own parser, which will be easier to use.  There is some interest in publishing my results, but please don't hold your breath :)   The keyservers do almost no validation, so you can find basically everything in there.

**Comment by cfcs** [comment]
@lambdafu: Have you looked into the Keybase database?

**Comment by lambdafu** [comment]
@cfcs Nope. Keybase is an interesting topic, but there is so much else to do right now. There is an API at https://keybase.io/docs/api/1.0 but I don't even know if there are dumps or if there is rate throttling.
